guides/common/modules/con_content-view-environment-ordering-and-priority.adoc

```diff
@@ -0,0 +1,62 @@
+[id="content_view_environment_ordering_and_priority_{context}"]
+= Content view environment ordering and priority
+
+When you assign multiple content view environments to a host or activation key, the order you specify determines the priority of the content.
+An activation key with multiple content view environments assigns them to hosts during registration in the same order as they are stored on the activation key.
+
+.Importance of content view environment order
+The order of assigned content view environments is critical in scenarios involving repository conflicts.
+A conflict occurs when two or more content view environments contain a repository with the same repository label.
+For example:
+A host assigned to the content view environments `Library/cv1, dev/cv1` uses the following configuration:
+[options="nowrap" subs="+quotes"]
+----
+hammer host update --name rhel9.example.com --organization-id 1 \
+--content-view-environments “Library/cv1, dev/cv1”
+----
+Both content view environments include the `satellite-client-6-for-rhel-9-x86_64-rpms` repository.
```

Comment:
What is the purpose of this RPM package? Is that a meta RPM package for Satellite Clients? What is the upstream equivalent?

Reply:
It's just being used as an example. The upstream equivalent is the Foreman Client repo; it includes katello-host-tools, tracer, yggdrasil, etc.